src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -127,6 +128,9 @@ public function iProceedSelectingShippingMethod($shippingMethod)
 
         $checkoutShippingPage = $this->getPage('Checkout\CheckoutShippingStep');
         $checkoutShippingPage->selectShippingMethod($shippingMethod);
+
+        $checkoutPaymentPage = $this->getPage('Checkout\CheckoutPaymentStep');
+        $checkoutPaymentPage->selectPaymentMethod('Offline');
```

Comment:
See (this comment)[https://github.com/Sylius/Sylius/pull/4010#discussion_r51862971]